src/OrchardCore.Modules/OrchardCore.Contents/Razor/ContentRazorHelperExtensions.cs

```diff
@@ -46,26 +46,26 @@ public static async Task<ContentItem> GetContentItemByHandleAsync(this IOrchardH
     /// </summary>
     /// <param name="orchardHelper">The <see cref="IOrchardHelper"/>.</param>
     /// <param name="contentItemId">The content item id to load.</param>
-    /// <param name="latest">Whether a draft should be loaded if available. <c>false</c> by default.</param>
+    /// <param name="option">The version to load. By default we load the Published version.</param>
     /// <example>GetContentItemByIdAsync("4xxxxxxxxxxxxxxxx").</example>
     /// <returns>A content item with the specific id, or <c>null</c> if it doesn't exist.</returns>
-    public static Task<ContentItem> GetContentItemByIdAsync(this IOrchardHelper orchardHelper, string contentItemId, bool latest = false)
+    public static Task<ContentItem> GetContentItemByIdAsync(this IOrchardHelper orchardHelper, string contentItemId, VersionOptions option = null)
```

Comment:
`= null`, why not the actual default value that is documented? I know this is what will be used internally but that would make more sense. If the other default changes for instance.

Reply:
VersionOptions is a class. How can we provide a default value to the argument?